subsys/caf/modules/sensor_sampler.c

```diff
@@ -15,21 +15,31 @@
 
 #define MODULE sensor_sampler
 #include <caf/events/module_state_event.h>
+#include <caf/events/power_event.h>
+#include <caf/events/power_manager_event.h>
 
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_CAF_SENSOR_SAMPLER_LOG_LEVEL);
 
 #define SAMPLE_THREAD_STACK_SIZE	CONFIG_CAF_SENSOR_SAMPLER_THREAD_STACK_SIZE
 #define SAMPLE_THREAD_PRIORITY		CONFIG_CAF_SENSOR_SAMPLER_THREAD_PRIORITY
 
+/* PM_DEVICE_STATE_ACTIVE should equal zero.
+ * Checking it just to be sure. As if anything changes in the code it would lead
```

Comment:
I think the first the leading line is enough. The more text we have the more we need to maintain.

Reply:
ok